test/dco.test.js

```diff
@@ -1,517 +1,6196 @@
+/*
+* The tests in this file verify the functionality of Probot DCO. Because there are many ways to
+* write a signoff and many ways to misconfigure git, the goal is to flush out as many corner cases
+* as possible.
+*
+* If you are adding a new feature, please note that you might need to repeat your test a few times
+* to verify that functionality is unchanged when various config options are enabled.
+*
+* Usage (from root directory): ./node_modules/jest/bin/jest.js
+* To see a list of tests: ./node_modules/jest/bin/jest.js --verbose
```

Comment:
I think it's safe to expect people have `npx` now, as it comes with all currently supported node versions. So folks can run `npx jest` or `npx jest --verbose`, which will run the same way on windows.

Reply:
Thanks! It's a good point, I'll push a commit that fixes this.